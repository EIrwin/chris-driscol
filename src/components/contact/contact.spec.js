// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Component from './contact.component';

const fixtures = {
  chris: {},
  relay: {
    environment: {},
  },
};

it('renders correctly', () => {
  const tree = renderer.create(<Component {...fixtures} />).toJSON();
  expect(tree).toMatchSnapshot();
});
