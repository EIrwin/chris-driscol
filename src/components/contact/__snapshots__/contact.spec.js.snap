// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="contact home-section"
  id="contactme"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-12 text-center"
      >
        <h2
          className="section-heading"
        >
          Contact Me
        </h2>
        <p
          className="section-subheading"
        >
          I would love to hear from you!
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-lg-12"
      >
        <form
          name="sentMessage"
          noValidate={true}
        >
          <div
            className="row"
          >
            <div
              className="col-md-6"
            >
              <div
                className="form-group"
              >
                <input
                  className="form-control"
                  id="email"
                  onChange={[Function]}
                  placeholder="your email *"
                  required={true}
                  type="email"
                  value=""
                />
              </div>
              <div
                className="form-group"
              >
                <input
                  className="form-control"
                  id="name"
                  onChange={[Function]}
                  placeholder="your name *"
                  required={true}
                  type="text"
                  value=""
                />
              </div>
              <div
                className="form-group"
              >
                <input
                  className="form-control"
                  id="subject"
                  onChange={[Function]}
                  placeholder="subject *"
                  required={true}
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              className="col-md-6"
            >
              <div
                className="form-group"
              >
                <textarea
                  className="form-control"
                  id="message"
                  onChange={[Function]}
                  placeholder="this is where you say something.. *"
                  required={true}
                  rows="4"
                  value=""
                />
              </div>
            </div>
            <div
              className="clearfix"
            />
            <div
              className="text-center"
            >
              <button
                className="btn btn-xl"
                onClick={[Function]}
              >
                Send Message
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</section>
`;
